pyscraper/get_links_from_ep.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+
+from everypolitician import EveryPolitician
+
+def output_file(country, legislature, filename):
+    data = EveryPolitician().country(country).legislature(legislature)
+    output_filename = "../members/{0}.xml".format(filename)
+    fp = open(output_filename,"w")
+    fp.write('''<?xml version="1.0" encoding="ISO-8859-1"?>
+    <publicwhip>
+    ''')
+
+    for person in data.popolo().persons:
```

Comment:
I don't know if this returns sorted or not, but again it would be good if it did (though perhaps as these are auto-generated we should gitignore them so doesn't matter so much).